### PR TITLE
Add error redirects for return downloads

### PR DESCRIPTION
**Pull request by embarnard**
This PR aims to resolve this [sentry error](https://codeforamerica.sentry.io/issues/5049306715/?project=1880327&referrer=pivotal_plugin)

### CODE REVIEW

**Review by tofarr — APPROVED** [review]

File: app/controllers/state_file/questions/submission_pdfs_controller.rb [hunk 1]
```diff
@@ -3,10 +3,17 @@ module Questions
     class SubmissionPdfsController < QuestionsController
       def show
         @submission = current_intake.efile_submissions.find_by(id: params[:id])
-        error_redirect and return unless @submission.present?
+        error_redirect and return #unless @submission.present?
```
Comment (tahsinaislam) [inline app/controllers/state_file/questions/submission_pdfs_controller.rb:6]: should we take out this comment?

**Review by tahsinaislam — APPROVED** [review]

File: app/controllers/hub/state_file/efile_submissions_controller.rb [hunk 2]
```diff
@@ -48,6 +48,11 @@ def state_counts
 
       private
 
+      def error_redirect
+        flash[:alert] = "There was a problem generating the tax return pdf."
+        redirect_back(fallback_location: request.referer)
+      end
+
```
Comment (tahsinaislam) [inline app/controllers/hub/state_file/efile_submissions_controller.rb:55]: is this method used in the hub somewhere?

**Review by tahsinaislam — APPROVED** [review]

File: app/controllers/state_file/questions/submission_pdfs_controller.rb (hunk 1 above)
Comment (embarnard) [reply app/controllers/state_file/questions/submission_pdfs_controller.rb]: omg you're totally right I was just commenting that out for testing 🙈 thanks for catching that @tahsinaislam

File: app/controllers/hub/state_file/efile_submissions_controller.rb (hunk 2 above)
Comment (embarnard) [reply app/controllers/hub/state_file/efile_submissions_controller.rb:55]: Yes this is used in the `show_pdf` method above

**Pull request merged**